Bhoon-coding/BestEats
# Feature/BSTS #44 remote notification

- initial Firebase in AppDelegate, pod install Firebase
- UserNotification authOption setting in AppDelegate
- check & access FCM token
- changed type casting (forced -> optional) in tableView, CollectionView
- remote notification test done
- add settingsViewController (layout done) & tabBar Item
- changed turn on/ off notification way (toggle -> go to settings page)
- change all strings to enum to avoid typos, check notification auth
- badge count initialize when entered app
- change func name (setUp -> configure)
